pkgs/development/compilers/go/1.4.nix

```diff
@@ -107,10 +107,12 @@ stdenv.mkDerivation rec {
   installPhase = ''
     mkdir -p "$out/bin"
     export GOROOT="$(pwd)/"
-    export GOBIN="$out/bin"
+    export GOBIN="$(pwd)/bin"
     export PATH="$GOBIN:$PATH"
     cd ./src
     ./all.bash
+    cd ../
```

Comment:
The next line after that uses $(pwd). I guess I can "cp $(pwd)/../bin/* $out/bin" instead? Your call.